src/main/features/response/components/payment-intention/payment-option.ts

```diff
@@ -0,0 +1,92 @@
+import * as express from 'express'
+
+import { Paths, FullAdmissionPaths } from 'response/paths'
+import { Paths as PaymentIntentionPaths } from 'response/components/payment-intention/paths'
+
+import { ErrorHandling } from 'main/common/errorHandling'
+import { DefendantPaymentOption, DefendantPaymentType } from 'response/form/models/defendantPaymentOption'
+import { Form } from 'main/app/forms/form'
+import { FormValidator } from 'main/app/forms/validation/formValidator'
+import { User } from 'main/app/idam/user'
+import { DraftService } from 'services/draftService'
+import { FeatureToggleGuard } from 'main/app/guards/featureToggleGuard'
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Draft } from '@hmcts/draft-store-client'
+import { Claim } from 'main/app/claims/models/claim'
+import { FeatureToggles } from 'utils/featureToggles'
+
+export class PaymentOptionPage {
+  constructor (private admissionType: string) {}
+
+  buildRouter (path: string = ''): express.Router {
```

Comment:
should this have a default?

Reply:
I made it default since I thought that from the component point of view it should be possible to install that route in '/' (root) without any additional configuration.

However since this is unlikely scenario I would be OK to not have default value and require binding point to be provided.

Reply:
its soo specific to admissions I would recommend removing

Reply:
default value removed